Jenkinsfile

```diff
@@ -305,7 +305,9 @@ pipeline {
                         CreateCluster('selfhealing')
                         runTest('storage', 'selfhealing')
                         runTest('self-healing', 'selfhealing')
+                        runTest('self-healing-chaos', 'selfhealing')
                         runTest('operator-self-healing', 'selfhealing')
```

Comment:
no self-healing-advanced-chaos ?

Reply:
it doesn't seem to exist for PSMDB, only for PXC, chaos tests at the moment were created as pairs for pumba tests